Error: Interface 'Response<ResBody>' incorrectly extends interface 'Response'

When I compile from typescript I get this bug

```
node_modules/@types/express-serve-static-core/index.d.ts:505:18 - error TS2430: Interface 'Response<ResBody>' incorrectly extends interface 'Response'.
  Types of property 'locals' are incompatible.
    Type 'Record<string, any>' is missing the following properties from type 'i18nAPI': locale, __, __n, __mf, and 5 more.

505 export interface Response<ResBody = any> extends http.ServerResponse, Express.Response {
                     ~~~~~~~~

```

from my package.json 

```
    "dependencies": {
        "@types/i18n": "^0.8.6",
        "@types/jest": "^26.0.7",
        "@types/moment": "^2.13.0",
        "@types/node": "^14.0.26",
        "@types/node-schedule": "^1.3.0",
        "@types/ws": "^7.2.6",
        "@typescript-eslint/eslint-plugin": "^3.7.0",
        "@typescript-eslint/parser": "^3.7.0",
        "@types/express": "^4.17.7",
        "chokidar": "^3.4.1",
        "dblapi.js": "^2.4.0",
        "discord.js": "^12.2.0",
        "eslint": "^7.5.0",
        "eslint-plugin-jest": "^23.18.2",
        "express": "^4.17.1",
        "findit2": "^2.2.3",
        "i18n": "^0.10.0",
        "jest": "^26.1.0",
        "moment": "^2.27.0",
        "node-schedule": "^1.3.2",
        "npm-check-updates": "^7.0.3",
        "tree-kill": "^1.2.2",
        "ts-jest": "^26.1.4",
        "typescript": "^3.9.7"
    }
```

Anyone know what's wrong?

--- a/types/express-serve-static-core/express-serve-static-core-tests.ts
+++ b/types/express-serve-static-core/express-serve-static-core-tests.ts
@@ -1,6 +1,13 @@
 import * as express from 'express-serve-static-core';
 
-const app: express.Application = {} as any;
+const app: express.Application<{
+    aKey: 'aValue'
+}> = {} as any;
+
+// App.locals can be extended
+app.locals.aKey; // $ExpectType "aValue"
+app.locals.bKey; // $ExpectError
+
 app.listen(3000);
 app.listen(3000, () => {
     // no-op error callback

--- a/types/express-serve-static-core/index.d.ts
+++ b/types/express-serve-static-core/index.d.ts
@@ -1044,7 +1044,9 @@ export type ApplicationRequestHandler<T> = IRouterHandler<T> &
     IRouterMatcher<T> &
     ((...handlers: RequestHandlerParams[]) => T);
 
-export interface Application extends EventEmitter, IRouter, Express.Application {
+export interface Application<
+    Locals extends Record<string, any> = Record<string, any>,
+> extends EventEmitter, IRouter, Express.Application {
     /**
      * Express instance itself is a request handler, which could be invoked without
      * third argument.
@@ -1210,7 +1212,7 @@ export interface Application extends EventEmitter, IRouter, Express.Application 
 
     map: any;
 
-    locals: Record<string, any>;
+    locals: Locals;
 
     /**
      * The app.routes object houses all of the routes defined mapped by the

--- a/types/express-serve-static-core/index.d.ts
+++ b/types/express-serve-static-core/index.d.ts
@@ -1045,7 +1045,7 @@ export type ApplicationRequestHandler<T> = IRouterHandler<T> &
     ((...handlers: RequestHandlerParams[]) => T);
 
 export interface Application<
-    Locals extends Record<string, any> = Record<string, any>,
+    Locals extends Record<string, any> = Record<string, any>
 > extends EventEmitter, IRouter, Express.Application {
     /**
      * Express instance itself is a request handler, which could be invoked without

--- a/types/express-serve-static-core/ts4.0/express-serve-static-core-tests.ts
+++ b/types/express-serve-static-core/ts4.0/express-serve-static-core-tests.ts
@@ -1,6 +1,13 @@
 import * as express from 'express-serve-static-core';
 
-const app: express.Application = {} as any;
+const app: express.Application<{
+    aKey: 'aValue'
+}> = {} as any;
+
+// App.locals can be extended
+app.locals.aKey; // $ExpectType "aValue"
+app.locals.bKey; // $ExpectError
+
 app.listen(3000);
 app.listen(3000, () => {
     // no-op error callback

--- a/types/express-serve-static-core/ts4.0/index.d.ts
+++ b/types/express-serve-static-core/ts4.0/index.d.ts
@@ -954,7 +954,9 @@ export type ApplicationRequestHandler<T> = IRouterHandler<T> &
     IRouterMatcher<T> &
     ((...handlers: RequestHandlerParams[]) => T);
 
-export interface Application extends EventEmitter, IRouter, Express.Application {
+export interface Application<
+    Locals extends Record<string, any> = Record<string, any>,
+> extends EventEmitter, IRouter, Express.Application {
     /**
      * Express instance itself is a request handler, which could be invoked without
      * third argument.
@@ -1120,7 +1122,7 @@ export interface Application extends EventEmitter, IRouter, Express.Application 
 
     map: any;
 
-    locals: Record<string, any>;
+    locals: Locals;
 
     /**
      * The app.routes object houses all of the routes defined mapped by the

--- a/types/express-serve-static-core/ts4.0/index.d.ts
+++ b/types/express-serve-static-core/ts4.0/index.d.ts
@@ -955,7 +955,7 @@ export type ApplicationRequestHandler<T> = IRouterHandler<T> &
     ((...handlers: RequestHandlerParams[]) => T);
 
 export interface Application<
-    Locals extends Record<string, any> = Record<string, any>,
+    Locals extends Record<string, any> = Record<string, any>
 > extends EventEmitter, IRouter, Express.Application {
     /**
      * Express instance itself is a request handler, which could be invoked without

